gov_uk_dashboards/formatting/human_readable.py

```diff
@@ -1,23 +1,46 @@
 """Functions for converting values to human readable format"""
 
 
-def format_as_human_readable(value_to_format: float, *, prefix: str = "") -> str:
-    """Format a number as a human readable string
-    Appends bn, m, k as appropriate
-    Rounds number to 3dp
+from typing import Optional
+
+
+def format_as_human_readable(
+    value_to_format: float,
+    *,
+    prefix: str = "",
+    suffix: str = "",
+    decimal_places: Optional[int] = None,
```

Comment:
What's the reasoning behind using this optional import over the standard named parameters? And why does this parameter use it but not prefix and suffix?

Reply:
Oh I see! That makes perfect sense, I completely misunderstood it! Thanks for the clarification